### ISSUE TITLE
As a Tech Lead, I want to be able to run a task to check the recency and completeness of data in the database

### ISSUE THREAD

**Issue by alexsoble** [raft-tech/TANF-app#796]
**Description:**

This ticket relates to the Contingency Plan, specifically section 5.2, "Validation Data Testing". This describes how the team would test the validity of the data in the event of a major system outage. 

We may want to write one or more scripts that would perform this data validation automatically if needed, rather than having the OFA Tech Lead run custom SQL queries against the database.

The following data validation script from the TTA Smart Hub project can serve as an example: 

https://github.com/HHS/Head-Start-TTADP/blob/2aa0a4ecd0c988d6c3a83536b29a39adf2accc87/src/tools/dataValidation.js

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] As a Tech Lead, I want to be able to run a task to check the recency and completeness of data in the database

As part of the Acceptance Criteria, the Tech Lead may want to run the script against the development or staging site to observe the results.

**Comment by alexsoble** [comment]
@ADPennington thoughts on when to prioritize: "when there's some data in the database".

— post-ATO
— possibly post-MVP as well

**Comment by valcollignon** [comment]
> This is something to firm up for Release 3 (when we have a backend). cc: @valcollignon

Adding to Release 3

**Comment by ADPennington** [comment]
We will revisit this topic in a new issue after Kibana is set up (after #1348 closes).
